bundles/org.openhab.binding.bluetooth.bluegiga/src/main/java/org/openhab/binding/bluetooth/bluegiga/internal/BlueGigaSerialHandler.java

```diff
@@ -183,6 +197,7 @@ public void close() {
      */
     public void close(long timeout) {
         close = true;
+        resetTransactionTimer();
         executor.shutdownNow();
         if (timerTask != null) {
             timerTask.cancel();
```

Comment:
isn't timeTask already null by this point? `resetTransactionTimer()` did this already.